vllm/spec_decode/draft_model_runner.py

```diff
@@ -108,13 +108,14 @@ def update_model_input(
                 seq.append_token_id(token_id, token_logprob.logprob)
                 seq.update_num_computed_tokens(1)
 
-        return self.prepare_model_input(self.cached_seq_group_metadata_list)
+        return self.prepare_model_input(self.cached_seq_group_metadata_list, 0)
```

Comment:
`virtual_engine` argument is not used?

Reply:
It's due to ABC and default args. Let me see if I can find some way around this.

Reply:
Done